apps/frontend/static_src/scss/main.scss

```diff
@@ -15,47 +16,34 @@
 @import "~bootstrap/scss/root";
 @import "~bootstrap/scss/reboot";
 @import "~bootstrap/scss/type";
-//@import "~bootstrap/scss/images";
-@import "~bootstrap/scss/containers";
-@import "~bootstrap/scss/grid";
-//@import "~bootstrap/scss/tables";
 @import "~bootstrap/scss/forms";
 @import "~bootstrap/scss/buttons";
 @import "~bootstrap/scss/transitions";
 @import "~bootstrap/scss/dropdown";
-// @import "~bootstrap/scss/button-group";
-@import "~bootstrap/scss/nav";
-//@import "~bootstrap/scss/navbar";
-//@import "~bootstrap/scss/card";
-//@import "~bootstrap/scss/accordion";
-//@import "~bootstrap/scss/breadcrumb";
-//@import "~bootstrap/scss/pagination";
-//@import "~bootstrap/scss/badge";
-//@import "~bootstrap/scss/alert";
-//@import "~bootstrap/scss/progress";
-//@import "~bootstrap/scss/list-group";
-//@import "~bootstrap/scss/close";
-//@import "~bootstrap/scss/toasts";
 @import "~bootstrap/scss/modal";
 @import "~bootstrap/scss/tooltip";
-//@import "~bootstrap/scss/popover";
-//@import "~bootstrap/scss/carousel";
-//@import "~bootstrap/scss/spinners";
-//@import "~bootstrap/scss/offcanvas";
-//@import "~bootstrap/scss/placeholders";
-
-// Helpers
-// @import "~bootstrap/scss/helpers";
 
 // Utilities
 @import "~bootstrap/scss/utilities/api";
 
 //custom component styles
-@import "./components/section_grid_block";
+@import "./components/app";
+@import "./components/section-grid-block";
 @import "./components/navigation";
-@import "./components/nav_button.scss";
+@import "./components/nav-button.scss";
 @import "./components/footer.scss";
 @import "./components/rich-text.scss";
 @import "./components/alert.scss";
-@import "./components/section_link";
-@import "./components/feedback_emoji.scss";
+@import "./components/section-link";
+@import "./components/feedback-emoji.scss";
+@import "./components/base.scss";
+@import "./components/header.scss";
+@import "./components/headings.scss";
+@import "./components/streamfield.scss";
+@import "./components/link-block.scss";
+@import "./components/toc.scss";
+@import "./components/dropdown-toggle.scss";
+@import "./components/search.scss";
+@import "./components/grid.scss";
+@import "./components/language-selector.scss";
+@import "./components/skip-link.scss";
```

Comment:
For consistency, could we either always have the `.scss` suffix or never?

Reply:
Fixed in bootstrap evaluation branch